client/src/components/organisms/DonorImpactSection.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FunctionComponent, useState } from "react";
+import Col from 'react-bootstrap/Col'
+import Row from 'react-bootstrap/Row'
+
+import MapWithMarkers, { MapWithMarkersPoint } from '../atoms/MapWithMarkers'
+import { Statistic, Testimonial } from '../../data/types/donorHomepageConfig'
+import BaseMap from '../../assets/kw-region-map.png'
+import Card from '../atoms/CardWithShadow'
+import Cursor from '../../assets/cursor.png'
+import DonorHomepageConfig from '../../config/donorHompageConfig.json'
+
+const DonorImpactSection: FunctionComponent = () => {
+  const [selectedTestimonial, _setSelectedTestimonial] = useState<Testimonial | null>(null);
+  const testimonials = DonorHomepageConfig.map.testimonials;
+  const setSelectedTestimonial = (id: number | null) => {
+    if (!id) {
+      _setSelectedTestimonial(null);
+      return;
+    }
+    const testimonial = testimonials.find((testimonial: Testimonial) => testimonial.id === id);
+    _setSelectedTestimonial(testimonial ?? null);
+  }
+  const points = DonorHomepageConfig.map.points;
+  const markerSize = testimonials.length > 0 && testimonials.length <= DonorHomepageConfig.map.markerSizes.length
+    ? DonorHomepageConfig.map.markerSizes[testimonials.length - 1]
+    : DonorHomepageConfig.map.defaultMarkerSize;
+  const maxPoints = points.length;
+  const testimonialPoints = testimonials.reduce(
```

Comment:
Unsure what this is supposed to do. Maybe we should store these values in the map object as we use them here?

Reply:
it transforms testimonials into points
hence `testimonialPoints`

I'll type the variables to be more clear